modules/client-configuration/pages/clients-slemicro.adoc

```diff
@@ -136,14 +153,14 @@ To add the client tools, add these channels from the command prompt:
 | OS Version
 | Client Channel
 
-| {micro} 5.3
-| sle-micro-5.3-uyuni-client
+| {micro} 5.1
+| suse-microos-5.1-uyuni-client
```

Comment:
Yes, `suse-microos` is right, that's the name for the SCC channels as well, for both 5.1 and 5.2